distro/adaptation/fedora/default

```diff
@@ -364,6 +365,7 @@ nbd-client: nbd
 nbd-server: nbd
 ncurses-bin: ncurses
 netcat-openbsd: nmap-ncat
+netpipe-lam:
 netpipe-lam: NetPIPE
```

Comment:
thanks, the above 2 lines looks wrong, both mapping to netpipe-lam. And help check failure of Checks '[Ruby / test-packages (fedora/41) (pull_request)](https://github.com/intel/lkp-tests/actions/runs/13825673390/job/38680399383?pr=492)"